### PR TITLE
Switching to contiguous webidl

### LINKED ISSUE #216: Please use Contiguous IDL instead

### PR DESCRIPTION
**Pull request by martinthomson**
Closes #216

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -571,18 +571,13 @@
...
+      <pre class="idl">
+interface PushManager {
+    Promise&lt;PushSubscription&gt; subscribe(optional PushSubscriptionOptionsInit options);
+    Promise&lt;PushSubscription?&gt; getSubscription();
+    Promise&lt;PushPermissionState&gt; permissionState(optional PushSubscriptionOptionsInit options);
+};
+</pre>
       <p>
         The <code><dfn id=
         "widl-PushManager-subscribe-Promise-PushSubscription--PushSubscriptionOptionsInit-options">subscribe</dfn></code>
```
Comment (beverloo) [inline index.html:583]: Here and elsewhere: the names of the members [aren't being linkified](https://cdn.rawgit.com/martinthomson/push-api/contiguous-webidl/index.html#extensions-to-the-serviceworkerregistration-interface) anymore. It looks like the definitions of the members need to indicate that they are, so this `<code>...</code>` should change to:

`<dfn for="PushManager" title="subscribe"><code>subscribe</code></dfn>`

(modulo #58, so in whichever order you prefer)

**Comment by marcoscaceres** [comment]
If you merge this in, I can help clean it up a bit more. There is a bunch of neat things ReSpec can do to link things together: http://github.com/w3c/respec/wiki/WebIDL-Guide

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:583]: This suggestion not not quite correct. You need to add `data-dfn-for="NameOfInterface"` on the section element. 

Again, let's just merge this an I can fix it all up.

Comment (beverloo) [reply index.html:583]: OK, let's do it.

Thanks for your help, Marcos! I'll make sure that I read up on the ReSpec WebIDL guide.

**Comment by marcoscaceres** [comment]
ok, on it! hope to have a PR in the next hour or so...

**Pull request merged**